testsuite/integration-arquillian/HOW-TO-RUN.md

```diff
@@ -44,6 +44,23 @@ This can be achieved by add the `auth-server-quarkus` profile when running the t
 Unlike the "development" setup described above, this requires re-build the whole distribution
 after doing any change in the code.
 
+### Running tests using an embedded server
+
+For test driven development, it is possible to run the Keycloak server deployed on real Quarkus server.
+This can be achieved by add the `auth-server-quarkus-embedded` profile when running the testsuite.
+
+    mvn -f testsuite/pom.xml -Pauth-server-quarkus-embedded clean install -Dtest=LoginTest
```

Comment:
Is it really necessary to build a whole /testsuite module? For the auth servers, we build them from the integration-arquillian module.

```suggestion
    mvn -f testsuite/integration-arquillian/pom.xml -Pauth-server-quarkus-embedded clean install -Dtest=LoginTest
```